src/main/java/org/broadinstitute/hellbender/tools/PrintReads.java

```diff
@@ -27,13 +31,17 @@
 public final class PrintReads extends ReadWalker {
 
     @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME, shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME, doc="Write output to this file")
-    public File OUTPUT;
+    public String OUTPUT_STR;
```

Comment:
I think we should just rename this to just `output`.  The all caps is wrong since it's not really constant.

Reply:
done